src/main/resources/changelog/changes/v3.2/create_instance_classification_table.xml

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.20.xsd">
+
+  <changeSet id="MSEARCH-667@@create_instance_classification_table" author="Pavlo_Smahin">
+    <preConditions>
+      <not>
+        <tableExists tableName="instance_classification"/>
+      </not>
+    </preConditions>
+
+    <comment>Create instance_classification table</comment>
+
+    <createTable tableName="instance_classification">
+      <column name="classification_type" type="VARCHAR(255)">
```

Comment:
"classification_type_id"? And should it be VARCHAR(255)?

Reply:
We can reduce VARCHAR size to UUID-size but it can't be changed to UUID type. As this field is nullable from business logic but can't be null in database as it is part of PK, I use `<null>` replacement for it.